### ISSUE TITLE
Add option: 'Always mute instead of block'

### ISSUE THREAD

**Issue by jsha** [jsha/blocktogether#11]

**Comment by jacnoc** [comment]
Came here to suggest this feature. Discovered it was already suggested (wow back in 2014!).

Would love to see this added! Blocking someone gives them the satisfaction of knowing you blocked them. Muting them leaves them "shouting into the void".

**Comment by mateohhh** [comment]
Yes, please add this. I'd like to automatically mute accounts instead of blocking them as well.

**Comment by wchristian** [comment]
Since it's been a while, for various reasons blocking is 100% useless to me. Any chance of allowing muting instead?

**Comment by dankegel** [comment]
I asked a science communicator whether a blocklist would be useful.  She said no, she muted instead.

I think it may be time.

**Comment by WojciechMigda** [comment]
I would like to have an option to block AND mute. Recently twitter started showing me tweets from accounts on my block list when I view trending topics and only augmenting my protection by additional mute gives me what I need.

**Comment by WojciechMigda** [comment]
If such a feature would be too much work maybe someone could create a script (python?) which would apply 'mute' to every accounts on my block list.
